manifests/init.pp

```diff
@@ -1,114 +1,56 @@
-  $broadcastclient        = $ntp::params::broadcastclient,
-  $config                 = $ntp::params::config,
-  $config_dir             = $ntp::params::config_dir,
-  $config_file_mode       = $ntp::params::config_file_mode,
-  $config_template        = $ntp::params::config_template,
-  $disable_auth           = $ntp::params::disable_auth,
-  $disable_dhclient       = $ntp::params::disable_dhclient,
-  $disable_kernel         = $ntp::params::disable_kernel,
-  $disable_monitor        = $ntp::params::disable_monitor,
-  $fudge                  = $ntp::params::fudge,
-  $driftfile              = $ntp::params::driftfile,
-  $leapfile               = $ntp::params::leapfile,
-  $logfile                = $ntp::params::logfile,
-  $iburst_enable          = $ntp::params::iburst_enable,
-  $keys                   = $ntp::params::keys,
-  $keys_enable            = $ntp::params::keys_enable,
-  $keys_file              = $ntp::params::keys_file,
-  $keys_controlkey        = $ntp::params::keys_controlkey,
-  $keys_requestkey        = $ntp::params::keys_requestkey,
-  $keys_trusted           = $ntp::params::keys_trusted,
-  $minpoll                = $ntp::params::minpoll,
-  $maxpoll                = $ntp::params::maxpoll,
-  $package_ensure         = $ntp::params::package_ensure,
-  $package_manage         = $ntp::params::package_manage,
-  $package_name           = $ntp::params::package_name,
-  $panic                  = $ntp::params::panic,
-  $peers                  = $ntp::params::peers,
-  $preferred_servers      = $ntp::params::preferred_servers,
-  $restrict               = $ntp::params::restrict,
-  $interfaces             = $ntp::params::interfaces,
-  $interfaces_ignore      = $ntp::params::interfaces_ignore,
-  $servers                = $ntp::params::servers,
-  $service_enable         = $ntp::params::service_enable,
-  $service_ensure         = $ntp::params::service_ensure,
-  $service_manage         = $ntp::params::service_manage,
-  $service_name           = $ntp::params::service_name,
-  $service_provider       = $ntp::params::service_provider,
-  $stepout                = $ntp::params::stepout,
-  $step_tickers_file      = $ntp::params::step_tickers_file,
-  $step_tickers_template  = $ntp::params::step_tickers_template,
-  $tinker                 = $ntp::params::tinker,
-  $tos                    = $ntp::params::tos,
-  $tos_minclock           = $ntp::params::tos_minclock,
-  $tos_minsane            = $ntp::params::tos_minsane,
-  $tos_floor              = $ntp::params::tos_floor,
-  $tos_ceiling            = $ntp::params::tos_ceiling,
-  $tos_cohort             = $ntp::params::tos_cohort,
-  $udlc                   = $ntp::params::udlc,
-  $udlc_stratum           = $ntp::params::udlc_stratum,
-  $ntpsigndsocket         = $ntp::params::ntpsigndsocket,
-  $authprov               = $ntp::params::authprov,
+  Boolean $broadcastclient,
+  Tea::Absolutepath $config,
+  Optional[Tea::Absolutepath] $config_dir,
```

Comment:
interesting, but yes I'd have expected those in core or stdlib - early days though